doc/src/documentation_style_guide.rst

```diff
@@ -0,0 +1,1019 @@
+
+[R340]https://en.wikipedia.org/wiki/Airy_function
+
+Sample Docstring
+================
+
+Here is an example of a correctly formatted docstring::
+
+  class gamma(Function):
+      r"""
+      The gamma function
+
+      .. math::
+          \Gamma(x) := \int^{\infty}_{0} t^{x-1} e^{-t} \mathrm{d}t.
+
+      Explanation
+      ===========
+
+      The ``gamma`` function implements the function which passes through the
+      values of the factorial function (i.e., `\Gamma(n) = (n - 1)!`), when n is
+      an integer. More generally, `\Gamma(z)` is defined in the whole complex
+      plane except at the negative integers where there are simple poles.
+
+      Examples
+      ========
+
+      >>> from sympy import S, I, pi, oo, gamma
+      >>> from sympy.abc import x
+
+      Several special values are known:
+
+      >>> gamma(1)
+      1
+      >>> gamma(4)
+      6
+      >>> gamma(S(3)/2)
+      sqrt(pi)/2
+
+      The Gamma function obeys the mirror symmetry:
+
+      >>> from sympy import conjugate
+      >>> conjugate(gamma(x))
+      gamma(conjugate(x))
+
+      Differentiation with respect to x is supported:
+
+      >>> from sympy import diff
+      >>> diff(gamma(x), x)
+      gamma(x)*polygamma(0, x)
+
+      Series expansion is also supported:
+
+      >>> from sympy import series
+      >>> series(gamma(x), x, 0, 3)
+      1/x - EulerGamma + x*(EulerGamma**2/2 + pi**2/12) + x**2*(-EulerGamma*pi**2/12 +
+      polygamma(2, 1)/6 - EulerGamma**3/6) + O(x**3)
+
+      We can numerically evaluate the gamma function to arbitrary precision
+      on the whole complex plane:
+
+      >>> gamma(pi).evalf(40)
+      2.288037795340032417959588909060233922890
+      >>> gamma(1+I).evalf(20)
+      0.49801566811835604271 - 0.15494982830181068512*I
+
+      See Also
+      ========
+
+      lowergamma: Lower incomplete gamma function.
+      uppergamma: Upper incomplete gamma function.
+      polygamma: Polygamma function.
+      loggamma: Log Gamma function.
+      digamma: Digamma function.
+      trigamma: Trigamma function.
+      sympy.functions.special.beta_functions.beta: Euler Beta function.
+
+      References
+      ==========
+
+      .. [1] https://en.wikipedia.org/wiki/Gamma_function
+      .. [2] http://dlmf.nist.gov/5
+      .. [3] http://mathworld.wolfram.com/GammaFunction.html
+      .. [4] http://functions.wolfram.com/GammaBetaErf/Gamma/
+
+      """
+
+Docstrings for Classes that are Mathematical Functions
+======================================================
+
+SymPy is unusual in that it also has classes that are mathematical functions.
+The docstrings for classes that are mathematical functions should include
+details specific to this kind of class, as noted below:
+
+* The Explanation section should include a mathematical definition of the
+  function. This should use LaTeX math. Use `$$ for inline math
+  <https://github.com/sympy/sympy/wiki/SymPy-Documentation-Style-Guide#formatting-
```

Comment:
Links to the wiki should be updated to use Sphinx cross referencing syntax so they link to this document.

Reply:
Ok giving it a try